core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/StackStatusFinalizerAbstractFlowConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.sequenceiq.cloudbreak.core.flow2;
+
+import javax.inject.Inject;
+
+import com.sequenceiq.flow.core.FlowEvent;
+import com.sequenceiq.flow.core.FlowState;
+import com.sequenceiq.flow.core.config.AbstractFlowConfiguration;
+import com.sequenceiq.flow.core.config.FlowFinalizerCallback;
+
+public abstract class StackStatusFinalizerAbstractFlowConfig<S extends FlowState, E extends FlowEvent> extends AbstractFlowConfiguration<S, E> {
+
+    @Inject
+    private StackStatusFinalizer stackStatusFinalizer;
+
+    protected StackStatusFinalizerAbstractFlowConfig(Class<S> stateType, Class<E> eventType) {
+        super(stateType, eventType);
+    }
+
+    @Override
+    public FlowFinalizerCallback getFinalizerCallBack() {
```

Comment:
Can you make this method `final` so that nobody can override it in cloudbreak?

Reply:
sure